compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/tree/SyntaxKind.java

```diff
@@ -110,6 +113,10 @@
     WHILE_STATEMENT(1205),
     CALL_STATEMENT(1206),
     PANIC_STATEMENT(1207),
+    CONTINUE_STATEMENT(1209),
+    BREAK_STATEMENT(1210),
```

Comment:
next number should be 1208

Reply:
return statement has used 1208 supun should I use it anyway?

Reply:
Use the next immediate available number.